.github/workflows/test.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: [18.13.0, '*']
```

Comment:
Bumped the CI execution to latest 18 as we were hitting some limitations with the node test runner:
- https://github.com/search?q=repo%3Anodejs%2Fnode+ERR_TAP_LEXER_ERROR&type=issues
- https://github.com/netlify/angular-runtime/actions/runs/7008488677/job/19064901851

Which have been fixed since then.